packages/utils/src/types/ticket.ts

```diff
@@ -58,19 +58,21 @@ export class Ticket {
 
   static create(
     counterparty: Address,
-    challenge: Address,
+    challenge: PublicKey,
     epoch: UINT256,
     index: UINT256,
     amount: Balance,
     winProb: UINT256,
     channelIteration: UINT256,
     signPriv: Uint8Array
   ): Ticket {
+    const challengeBytes = challenge.toAddress()
+
     const hash = toEthSignedMessageHash(
       u8aToHex(
         serializeToU8a([
           [counterparty.serialize(), Address.SIZE],
-          [challenge.serialize(), Address.SIZE],
+          [challengeBytes.serialize(), Address.SIZE],
```

Comment:
shouldn't this be `PublicKey.SIZE` ?

Reply:
ah nvm, just noticed it gets converted to an address, the name `challengeBytes` doesn't seem applicable though, lets find a better name

Reply:
changed, now called `encodedChallenge`